tests/_test_msui/test_msui.py

```diff
@@ -328,3 +336,181 @@ def test_flight_track_io(self, mockload, mocksave, mockq, mocki, mockw):
         assert self.window.listFlightTracks.count() == 2
         assert os.path.exists(self.save_ftml)
         os.remove(self.save_ftml)
+
+
+class Test_MSUIMainWindow:
+    def test_storing_and_restoring(self, qtbot, mswms_server):
+        """Test the full scenario: create flight track, open TopView, modify settings,
+        save on close, and restore settings on reopen."""
+
+        parsed_url = urllib.parse.urlparse(mswms_server)
+        scheme, host, port = parsed_url.scheme, parsed_url.hostname, parsed_url.port
+        server_url = f"{scheme}://{host}:{port}"
+        logging.debug(server_url)
+
+        window = msui_mw.MSUIMainWindow()
+        window.show()
+
+        # create 1st flighttrack
```

Comment:
it is maybe better to save to a filename so that flightpath file exists and to avoid storing anything for the current default "new flight track (...)"

Because when you do use it for temporary data at the moment someone hitting the create new button and having something in the json file like, 
```
{
  "new flight track (2)": {
    "global": {
      "mss_version": "10.1.0",
      "flight_track_name": "new flight track (2)"
    },
    "views": [
      {
        "view_type": "topview",
        "view_id": "view_topview_0"
      },
      {
   ```
      
   windows gets pushed on him   
   
   Changes on the behaviour should be done in the next PR